### PR TITLE
Feature/tailwind

### PR DESCRIPTION
**Pull request by firestar300**
Add some styles to components for presentation purpose.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import './css/style.css'
+import Tabs from './classes/Tabs'
```
Comment (Rahe) [inline src/index.js:2]: Codacy has a fix for the issue: [Replace `Tabs·from·'./classes/Tab` with `'./css/style.cs`](https://app.codacy.com/gh/BeAPI/accessible-components/pullRequest?prid=8000264)
```suggestion
import './css/style.css'
```

Comment (Rahe) [inline src/index.js:2]: Codacy has a fix for the issue: [Replace `Tabs·from·'./classes/Tab` with `'./css/style.cs`](https://app.codacy.com/gh/BeAPI/accessible-components/pullRequest?prid=8000264)
```suggestion
import './css/style.css'
```

Comment (Rahe) [inline src/index.js:2]: Codacy has a fix for the issue: [Replace `Tabs·from·'./classes/Tab` with `'./css/style.cs`](https://app.codacy.com/gh/BeAPI/accessible-components/pullRequest?prid=8000264)
```suggestion
import './css/style.css'
```

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import './css/style.css'
```
Comment (Rahe) [inline src/index.js:1]: Codacy has a fix for the issue: [Replace `'./css/style.cs` with `Tabs·from·'./classes/Tab`](https://app.codacy.com/gh/BeAPI/accessible-components/pullRequest?prid=8000264)
```suggestion
import Tabs from './classes/Tabs'
```

**Review by francoistibo — APPROVED** [review]

**Pull request merged**